dbt_style_guide.md

```diff
@@ -1,237 +1,508 @@
-        end as cancellation_date,
+- Joins should list the left table first (i.e., the table you're joining data to)  
+  Example:
+  ```sql
+  select
+      trips.*,
+      drivers.rating as driver_rating,
+      riders.rating as rider_rating
+  from trips
+  left join users as drivers
+     on trips.driver_id = drivers.user_id
+  left join users as riders
+      on trips.rider_id = riders.user_id
 
-        some_cte_agg.total_field_4,
-        some_cte_agg.max_field_5
+  ```
 
-    from my_data
-    left join some_cte_agg  
-        on my_data.id = some_cte_agg.id
-    where my_data.field_1 = 'abc'
-        and (
-            my_data.field_2 = 'def' or
-            my_data.field_2 = 'ghi'
-        )
-    having count(*) > 1
+### Example SQL
+  ```sql
+  with
+
+  my_data as (
+      select * from {{ ref('my_data') }}
+      where not is_deleted
+  ),
+
+  some_cte as (
+      select * from {{ ref('some_cte') }}
+  ),
+
+  some_cte_agg as (
+      select
+          id,
+          sum(field_4) as total_field_4,
+          max(field_5) as max_field_5
+      from some_cte
+      group by 1
+  ),
+
+  final as (
+      select [distinct]
+          my_data.field_1,
+          my_data.field_2,
+          my_data.field_3,
+
+          -- use line breaks to visually separate calculations into blocks
+          case
+              when my_data.cancellation_date is null
+                  and my_data.expiration_date is not null
+                  then expiration_data
+              when my_data.cancellation_date is null
+                  then my_data.start_date + 7
+              else my_data.cancellation_date
+          end as cancellation_date,
+
+          some_cte_agg.total_field_4,
+          some_cte_agg.max_field_5
+      from my_data
+      left join some_cte_agg  
+          on my_data.id = some_cte_agg.id
+      where 
+          my_data.field_1 = 'abc'
+          and (
+              my_data.field_2 = 'def'
+              or my_data.field_2 = 'ghi'
+          )
+      qualify row_number() over(
+          partition by my_data.field_1
+          order by my_data.start_date desc
+      ) = 1
+  )
 
-)
+  select * from final
+  ```
 
-select * from final
+## YAML and Markdown style guide
 
-```
+- Every subdirectory contains their own `.yml` file(s) which contain configurations for the models within the subdirectory.
 
-- Your join should list the "left" table first (i.e. the table you are selecting `from`):
-```sql
-select
-    trips.*,
-    drivers.rating as driver_rating,
-    riders.rating as rider_rating
+- YAML and markdown files should be prefixed with an underscore ( `_` ) to keep it at the top of the subdirectory.
 
-from trips
-left join users as drivers
-    on trips.driver_id = drivers.user_id
-left join users as riders
-    on trips.rider_id = riders.user_id
+- YAML and markdown files should be named with the convention `_<description>_<config>`.  
+  Examples: `_jaffle_shop_sources.yml`, `_jaffle_shop_docs.md`  

```

Comment:
```suggestion
  Examples: `_jaffle_shop__sources.yml`, `_jaffle_shop__docs.md`  

```

Reply:
Good catch! Will get these in on Monday!

Reply:
[Done](https://github.com/dbt-labs/corp/pull/63/commits/bcb93bf8b7401ff3cc7de9682bcbe20cb0bf6642). Thank you!